server/channels/app/user.go

```diff
@@ -1396,9 +1398,9 @@ func (a *App) sendUpdatedUserEvent(user *model.User) {
 	message.GetBroadcast().ContainsSanitizedData = true
 	a.Publish(message)
 
-	// send unsanitized user to event creator
-	sourceUserMessage := model.NewWebSocketEvent(model.WebsocketEventUserUpdated, "", "", unsanitizedCopyOfUser.Id, nil, "")
-	sourceUserMessage.Add("user", unsanitizedCopyOfUser)
+	sourceUserCopyOfUser.Sanitize(nil)
```

Comment:
The various `*Sanitize*` APIs are all over the place. This matches the way we're returning data to the user via the REST API. I've avoided touching the broad surface area of these APIs on purpose.